src/editing_tools/CropTool.vala

```diff
@@ -1064,41 +1096,45 @@ public class EditingTools.CropTool : EditingTool {
             height = bottom - top + 1;
 
             switch (in_manipulation) {
-            case BoxLocation.LEFT_SIDE:
-            case BoxLocation.TOP_LEFT:
-            case BoxLocation.BOTTOM_LEFT:
-                if (width < CROP_MIN_SIZE)
-                    left = right - CROP_MIN_SIZE;
-                break;
-
-            case BoxLocation.RIGHT_SIDE:
-            case BoxLocation.TOP_RIGHT:
-            case BoxLocation.BOTTOM_RIGHT:
-                if (width < CROP_MIN_SIZE)
-                    right = left + CROP_MIN_SIZE;
-                break;
-
-            default:
-                break;
+                case BoxLocation.LEFT_SIDE:
+                case BoxLocation.TOP_LEFT:
+                case BoxLocation.BOTTOM_LEFT:
+                    if (width < CROP_MIN_SIZE) {
+                        left = right - CROP_MIN_SIZE;
+                    }
+                    break;
+
+                case BoxLocation.RIGHT_SIDE:
+                case BoxLocation.TOP_RIGHT:
+                case BoxLocation.BOTTOM_RIGHT:
+                    if (width < CROP_MIN_SIZE) {
+                        right = left + CROP_MIN_SIZE;
+                    }
+                    break;
+
+                default:
```

Comment:
Elementary code style: blank line after `if (){}`
```suggestion
                    if (width < CROP_MIN_SIZE) {
                        right = left + CROP_MIN_SIZE;
                    }

                    break;
                default:
```
Many examples not additionally commented